aten/src/ATen/native/quantized/cpu/qfc.cpp

```diff
@@ -52,11 +50,15 @@ class QFCInt8 final : public c10::OperatorKernel {
     AT_ASSERT(bias.size(0) == N);
     AT_ASSERT(bias.dim() == 1);
 
-    float input_scale_float = static_cast<float>(input_scale);
-    int32_t input_zero_point_int32 = static_cast<int32_t>(input_zero_point);
+    float input_scale_float = static_cast<float>(input.q_scale().to<double>());
+    int32_t input_zero_point_int32 =
+        static_cast<int32_t>(input.q_zero_point().to<double>());
```

Comment:
Is there a reason to go to double and go to int32 after that? Can't we do `q_zero_point().toInt()`? or `.toLong()`?

Reply:
This issue has been addressed. Thanks for pointing it out!